.github/renovate.json

```diff
@@ -4,10 +4,9 @@
     ":rebaseStalePrs",
     ":pinDependencies",
     ":masterIssue",
-    ":timezone(Europe/Amsterdam)",
-    ":assignee(vidavidorra)",
-    "docker:enableMajor"
+    ":timezone(Europe/Amsterdam)"
   ],
+  "enabledManagers": ["npm", "regex"],
```

Comment:
I think this can be removed.

Reply:
I've had it like that in one of my repo's but think it didn't work. I might have done something else wrong there though, so I'll remove this and we'll find out in the next log :)